book/src/pong-tutorial/contribution.md

```diff
@@ -0,0 +1,13 @@
+# Call for contribution
+
+And... that's where the pong chapter ends.
+Amethyst is still in development and writing good tutorials is hard, so the guide is not complete yet.
+We are sorry this might feel a bit disappointing.
+
+Fortunately, the pong example is complete, and you can find the entire code with balls, score and music on the example pages available [here](https://www.amethyst.rs/doc). You can get additional help by leaving a post on [our subreddit](https://www.reddit.com/r/Amethyst) or on our Discord server.
+
+We hate to have an incomplete tutorial as much as you do, and you can help us complete it!
+As stated above, the code is ready, tested and working. The only thing we need now is good English.
+Even if you have limited experience with the engine, if you think you know how to write a tutorial, you can reach us out and we will help you do it by giving you details on how things work and how to structure it.
```

Comment:
Actually, "good enough" English skills and good knowledge of the engine would be preferable IMO. Probably both would help.